config/routes.rb

```diff
@@ -8,12 +8,15 @@
 
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
+  get 'users/:id' => 'users#show', as: :user
```

Comment:
I think it would b better to write this as:

    resources :users, only: [:show]

Since we could easily have more options such as edit, update, etc.